P4Plugin/Source/P4SpecCommand.cpp

```diff
@@ -10,49 +12,106 @@ class P4SpecCommand : public P4Command
 	{
 		ClearStatus();
 		m_Root.clear();
-		
+
+		Conn().Log().Info() << args[0] << "::Run()" << Endl;
+		std::string fallback_error = std::string();
```

Comment:
Don't init with std::string() it's already called by just creating the variable
Then, let's have the 3 lines in one
		`const std::string fallback_error = (args.size() > 1) ? args[1] : std::string();`

Reply:
Done!